### ISSUE TITLE
ModuleNotFoundError: No module named 'tensorflow.contrib'

### ISSUE THREAD

**Issue by TheCodingRecruiter** [minimaxir/gpt-2-simple#137]
I had ran the program initially and I didn't have tensorflow installed and so got a module error telling me that Tensorflow wasn't installed.  Then I pip installed it and now I am getting this error.  I googled this error, and have found out that TF 1.14 moved contrib to contrib.trt.  Since I just installed TF and have the most recent version, this may be why I am encountering the error, but I don't see it listed in the requirements txt on the repo, so I am unsure.  Can you assist?  For the record, I am trying to run this within pythonanywhere.

**Comment by WAUthethird** [comment]
You’re positive that you have Tensorflow 1, not 2 installed? GPT-2-Simple doesn’t support TF 2

**Comment by TheCodingRecruiter** [comment]
Nope.  That must be what it is.  I have TF 2.  I'll uninstall and install the right version.  What version is the last version? TF 1.13?

**Comment by WAUthethird** [comment]
It is TF 1.15.

**Comment by minimaxir** [comment]
Yes, that is a TF 2.0 warning, currently this package doesn't support it.

**Comment by mobyte0** [comment]
Why not add `tensorflow==1.14` to `requirements.txt`?

**Comment by minimaxir** [comment]
Because tensorflow has two flavors: CPU (`tensorflow`) and GPU (`tensorflow-gpu`) and I do not want to force one or the other.

**Comment by arnicas** [comment]
I just hit this too, and was coming to say add it to requiremnts :)  Oh well!

**Comment by minimaxir** [comment]
I am open to adding a hard assert if the tf package version >= `2.0.0`.

**Comment by minimaxir** [comment]
The assert is added; let me know if the copy needs to be more clear.

**Comment by arlyon** [comment]
What would it take to port this to 2.0? I have a project that already uses 2.0

**Comment by samcamwilliams** [comment]
This tripped me up, too. Apparently tensorflow 1.15 supports _both_ CPU and GPU execution. Is there no way that we can add `tensorflow==1.15` to the `dockerfile`?
